.pre-commit-hooks.yaml

```diff
@@ -4,8 +4,16 @@
   language: docker_image
   entry: returntocorp/semgrep:develop
 
-- id: semgrep
+- id: semgrep-docker
```

Comment:
Not sure why Github won't allow me to comment on the unchanged line above... Should we move `semgrep-develop` -> `semgrep-docker-develop` above to stay consistent here?